markdown/events/2021/bytesize-16-module-test-data.md

```diff
@@ -27,3 +28,40 @@ This week, Kevin Menden ([@KevinMenden](http://github.com/KevinMenden/)) will pr
+
+[4:54](https://youtu.be/QXfAerydAT0?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=294).. For this the most important file is the `test_data.config` file, which is stored in `modules/tests/config/test_data.config directory`. So this file is used to define all these different test data files that are available. It has a link to the test data directory, which is the nf-core test dataset repository, and specifically the modules branch of this repository where we store all the test data. There are links to specific files in this directory, and these links are again linked in this dictionary. This is just a small example of this whole test data parameter dictionary. The reason we do it like this is so that we are able to change these links at some point without changing the code in the `test_main.nf` files of the modules because you will only have these keys here and these entries are defined in your script, so you don’t really need to care about what’s happening on this side of the code. So this is where I can look for the keys and the data I need in the file.
+
+[6:12](https://youtu.be/QXfAerydAT0?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=372) So where can we get this data from? For this you need to look within the `modules` branch of the `nf-core/test-datasets` repository (<https://github.com/nf-core/test-datasets/tree/modules>). This repository is where we store all the datasets that we use for testing. There’s a `README` file, where we try to explain these different test datasets. Please make sure to take a look at that. Then also go through the repository and go through the datasets you need, to ensure that they are the correct ones.
+
+[6:52](https://youtu.be/QXfAerydAT0?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=412) We currently have mainly two genomes available, one is the sarscov2 and the other is a subset of the human chromosome 22. The reason for this is that sarscov2 is really small, so we can really use the whole genome, and use all kinds of files for this genome without having large datasets. This is important for two reasons, first, we want to have small datasets to test, so that the test repository does not explode and the tests are also smaller for small datasets. For the human genome viz. part of chromosome 22, we again have a small dataset. Currently we only have genomics data available on the test dataset repository, and we also have some non-standard genomics data for specific tools that have been added by users.
```

Comment:
```suggestion
[6:52](https://youtu.be/QXfAerydAT0?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=412) We currently have mainly two genomes available, one is the sarscov2 and the other is a subset of the human chromosome 22. The reason for this is that sarscov2 is really small, so we can really use the whole genome, and use all kinds of files for this genome without having large datasets. This is important for two reasons, first, we want to have small datasets to test, so that the test repository does not explode and the tests are also smaller for small datasets. For the human genome subset of chromosome 22, we again have a small dataset. Currently we only have genomics data available on the test dataset repository, and we also have some non-standard genomics data for specific tools that have been added by users.
```